x-pack/legacy/plugins/code/public/app.tsx

```diff
@@ -12,11 +12,7 @@ import { i18n } from '@kbn/i18n';
 import { I18nProvider } from '@kbn/i18n/react';
 import moment from 'moment';
 import { CoreStart } from 'src/core/public';
-import 'ui/autoload/all';
-import 'ui/autoload/styles';
```

Comment:
I'd suggest remove all angular related modules at once when you switch to ApplicationService https://github.com/restrry/kibana/blob/3977c7676e13dfa55b2953421d910daacbc2a7c4/rfcs/text/0004_application_service_mounting.md
It's already available on the client. Example from our tests https://github.com/restrry/kibana/blob/3977c7676e13dfa55b2953421d910daacbc2a7c4/test/plugin_functional/plugins/core_plugin_b/public/plugin.tsx#L38

Reply:
Add back these 2 items. And I will have a different PR to migrate to the new ApplicationService.